tests/Http/Controller/Reviewer/DashboardControllerTest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace OpenCFP\Test\Http\Controller\Reviewer;
+
+use OpenCFP\Test\WebTestCase;
+
+class DashboardControllerTest extends WebTestCase
+{
+    /**
+     * @test
+     */
+    public function indexActionWorksWithoutTalks()
```

Comment:
I can't quite figure out why this test is failing, as it works fine for me locally.